docs/guides/cloud.rst

```diff
@@ -16,6 +16,76 @@ EdgeDB Cloud is the easiest way to host your EdgeDB instance. We offer two ways
 to interact with EdgeDB Cloud: via our CLI or through a graphical web
 interface.
 
+.. edb:youtube-embed:: IG1MggUzzH4
+
+Usage
+=====
+
+
+1. Log in
+---------
+
+To use the CLI with EdgeDB Cloud, start by running
+:ref:`ref_cli_edgedb_cloud_login`. This will open a browser and allow you to
+log in to EdgeDB Cloud.
+
+.. note::
+
+    During the Cloud beta, you will only be able to successfully complete
+    authentication if you have been invited to the beta.
+
+
+2. Create an instance
+---------------------
+
+Once your login is complete, you may use the other CLI commands to create and
+interact with Cloud instances. To create an Cloud instance, you can use one of
+these commands:
+
+* :ref:`ref_cli_edgedb_instance_create` with an instance name of
+  ``<github-username>/<instance-name>``.
+
+  .. code-block:: bash
+
+      $ edgedb instance create <github-username>/<instance-name>
+
+* :ref:`ref_cli_edgedb_project_init` with the ``--server-instance`` option. Set
+  the server instance name to ``<github-username>/<instance-name>``.
+
+  .. code-block:: bash
+
+      $ edgedb project init \
+        --server-instance <github-username>/<instance-name>
+
+  Alternatively, you can run ``edgedb project init`` *without* the
+  ``--server-instance`` option and enter an instance name in the
+  ``<github-username>/<instance-name>`` format when prompted interactively.
+
+.. note::
+
+    Please be aware of the following restrictions on EdgeDB Cloud instance
+    names:
+
+    * can contain only Latin alpha-numeric characters or ``-``
+    * cannot start with a dash (``-``) or contain double dashes (``--``)
+    * maximum instance name length is 61 characters minus the length of your
+      GitHub username (i.e., length of GitHub username + length of instance
+      name must be fewer than 62 characters)
+
+
+
+3. Configure your application
```

Comment:
I wonder if we can/should just move this into a section below the CLI and Cloud interfaces sections, like:

```
Deploying your application
====================

```

WDYT?

Reply:
Yeah, I like that. You also pushed me to re-evaluate more broadly. A lot of what I brought over was already in what we had. I've pulled out what wasn't there from the old guide and better integrated it which allowed me to delete the whole "Usage" section altogether, alongside moving the deployment info. Does this look better?